web/src/lib/static/admin/findDeadLinks.ts

```diff
@@ -212,15 +223,31 @@ export const findDeadLinks = async (): Promise<Record<string, string[]>> => {
     ...filenames.filings.map((it) => {
       return `/filings/${it.split(".md")[0]}`;
     }),
+    "/welcome",
```

Comment:
This kinda stinks that populating pages is a manual process. I can see this getting looked over easily as other application changes are made.

I don't have a solution right now, but it'd be cool if there's some way that we could use logic from routes to initialize this variable.

Reply:
I totally second this and might be good as a refactor ticket in the back log for the future. I'll make one right now